src/ServicePulse.Host/app/js/app.constants.js

```diff
@@ -5,7 +5,7 @@
         .constant('showPendingRetry', false)
         .constant('scConfig', {
             service_control_url: 'http://localhost:33333/api/',
-            monitoring_urls: ['http://localhost:33633/']
+            monitoring_urls: ['http://localhost:1234/']
```

Comment:
Ok let us change default port of SC.Monitoring to be 33633, and let's revert this change.